cmd/bootstrapper/generator_test.go

```diff
@@ -140,12 +140,12 @@ func TestGenerator_Generate(t *testing.T) {
 			defer ts.Close()
 
 			fs := afero.NewMemMapFs()
+
 			g := NewGenerator(
 				ts.URL,
 				cfg.PublicListener,
 				WithLogger(zaptest.NewLogger(t)),
 				WithFilesystem(fs),
-				WithHttpClient(ts.Client()),
```

Comment:
I believe `WithHttpClient` is now unused and can be removed (it was only used in tests anyway).